metadata.json

```diff
@@ -64,13 +64,6 @@
       "operatingsystemrelease": [
         "18.04"
       ]
-    },
-    {
-      "operatingsystem": "FreeBSD",
-      "operatingsystemrelease": [
-        "12",
-        "13"
-      ]
```

Comment:
This should be noted somewhere. I think it'd also be good to have it in a separate commit so it isn't lost. Generally speaking we try to keep the modulesync commits completely autogenerated.

Reply:
Updated with second commit saying why